app/uk/gov/hmrc/economiccrimelevyregistration/forms/mappings/MaxLengths.scala

```diff
@@ -18,10 +18,12 @@ package uk.gov.hmrc.economiccrimelevyregistration.forms.mappings
 
 object MaxLengths {
 
-  val EmailMaxLength            = 132
-  val NameMaxLength             = 160
-  val TelephoneNumberMaxLength  = 24
-  val RoleMaxLength             = 160
-  val OrganisationNameMaxLength = 160
+  val EmailMaxLength                     = 132
+  val NameMaxLength                      = 160
+  val TelephoneNumberMaxLength           = 24
+  val RoleMaxLength                      = 160
+  val OrganisationNameMaxLength          = 160
+  val CharityRegistrationNumberMaxLength = 8
```

Comment:
I haven't found any documentation related to charity registration number (CHRN) constraints. Did you had chat with someone who confirmed that it should be 8 letters long ?

Reply:
Removed constraint on charity number length.